ekm/signer_key_manager_test.go

```diff
@@ -35,7 +34,7 @@ func testKeyManager(t *testing.T) spectypes.KeyManager {
 	db, err := getBaseStorage(logger)
 	require.NoError(t, err)
 
-	km, err := NewETHKeyManagerSigner(db, beacon2.NewNetwork(core.PraterNetwork, 0), types.GetDefaultDomain(), logger)
```

Comment:
add tests with blinded on?

Reply:
The behavior would be same. But I'll better pass `true` to test that it works well in this case. The tests here make sure that signing once works and signing twice is slashable